### ISSUE TITLE
GutterTalk 49-61 missing featured image banners

### ISSUE THREAD

**Issue by luetkemj** [luetkemj/mcdc#63]
They didn't come over in the original import. Looks like they just need to be reassigned.

**Comment by headcomic** [comment]
What does that mean? (Confused)

**Comment by luetkemj** [comment]
These posts work and show up on the gutter talk page properly. But they are missing the show art.

What they should look like:
![screen shot 2015-05-16 at 1 25 45 pm](https://cloud.githubusercontent.com/assets/925980/7667243/271bcd2a-fbcf-11e4-91c3-04585e4c79b6.png)

What they look like:
![screen shot 2015-05-16 at 1 25 50 pm](https://cloud.githubusercontent.com/assets/925980/7667244/2d19329e-fbcf-11e4-9603-b90b1f8ccb67.png)

These are set by assigning a featured image to the post. It looks like for some silly reason the featured image got lost in the import. We need to find the image in the media library and reassign it or find the originals and reupload them.

**Comment by headcomic** [comment]
Don't worry about fixing these, it is pretty simple as we have backups. Adam or I will do it. 

More concerning is whether or not there were any other images within other posts that were screwed up like this.

**Comment by luetkemj** [comment]
My guess is no. The reason I think this happened is they did not come over in the initial import. This is because they had no category attached to them I think. 

Typically when you import you import and download all images. I did not do that this time because the location of the images would not have made sense after we did the site swap stuff.

Because I did not do that, the featured image did not get set. I mistakenly believed that since the images existed on the server and the paths in the db should have pointed directly to them that the featured image would just work.

Hope that makes some sense.
